test_conformance/relationals/test_shuffles.cpp

```diff
@@ -619,38 +619,33 @@ int test_shuffle_dual_kernel(cl_context context, cl_command_queue queue,
         return error;
 
     typeSize = get_explicit_type_size( vecType );
+    std::vector<cl_long> inData(inVecSize * numOrders * sizeof(cl_long));
+    std::vector<cl_long> inSecondData(inVecSize * numOrders * sizeof(cl_long));
+    std::vector<cl_long> outData(outRealVecSize * numOrders * sizeof(cl_long));
```

Comment:
We want to pass the number of elements, not the numbre of bytes.

```suggestion
    std::vector<cl_long> inData(inVecSize * numOrders);
    std::vector<cl_long> inSecondData(inVecSize * numOrders);
    std::vector<cl_long> outData(outRealVecSize * numOrders);
```